src/menu/tray.js

```diff
@@ -10,7 +10,7 @@ const electron = require('electron');
 const k8s = require('@kubernetes/client-node');
```

Comment:
Why are you not using `import` stmts here?

Reply:
Pretty much because I haven't converted that file to TypeScript yet.

Reply:
Separate PR then?

Reply:
Yep, that will happen after this one; I didn't want to convert everything in one go due to the high likelihood of generating merge conflicts.